src/functions.js

```diff
@@ -1,9 +1,15 @@
 function getTitle( document ) {
+
   var element = document.getElementsByTagName('title')[0].innerHTML;
   return element;
 };
 
 function getNumberOfBikes( document ) {
+
+  var bikeCount = document.getElementById('donation-count-alert').innerHTML;
+  var numBikes = parseInt(bikeCount);
+  return numBikes;
```

Comment:
you can be even more concise here: 
If you declare a variable, and then return that variable straight away without doing anything with it, then you can just return the value you want to assign the variable. 
So
```
var numBikes = parseInt(bikeCount);
return numBikes;
```
can become a single line:
```
return parseInt(bikeCount);
```

Reply:
@robota-x thanks I do like the single line better